tests/pytest/test_logs.py

```diff
@@ -0,0 +1,66 @@
+import os
+import sys
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import cocotb
+import pytest
+from cocotb.clock import Clock
+from cocotb.runner import get_runner
+from cocotb.triggers import ClockCycles
+from test_cocotb import (
+    compile_args,
+    gpi_interfaces,
+    hdl_toplevel,
+    hdl_toplevel_lang,
+    sim_args,
+    sim_build,
+    tests_dir,
+    verilog_sources,
+    vhdl_sources,
+)
+
+sys.path.insert(0, os.path.join(tests_dir, "pytest"))
```

Comment:
Would be nicer to stay with `pathlib` instead of `os.path` but that's minor.

Reply:
This was copypastated from `test_waves.py`.  But I can take a look here.

Reply:
We are certainly not consistent ourselves yet.